operator/roles/forkliftcontroller/defaults/main.yml

```diff
@@ -22,6 +22,7 @@ controller_configmap_name: "{{ controller_service_name }}-config"
 controller_service_name: "{{ app_name }}-controller"
 ovirt_osmap_configmap_name: "forklift-ovirt-osmap"
 vsphere_osmap_configmap_name: "forklift-vsphere-osmap"
+virt_customize_configmap_name: "{{ app_name }}-virt-customize"
```

Comment:
maybe calling it `forklift-virt-customize `like the osmaps makes more sense because it will make the name deterministic, we will not need to look for different config file depending on the deployment, it will be the same name everywhere, WDYT ?